### ISSUE TITLE
Docker run in README.md doesn't works

### ISSUE THREAD

**Issue by fooock** [prysmaticlabs/prysm#3863]
Running via Docker, section Linux/Mac in [README.md](https://github.com/prysmaticlabs/prysm/blob/master/README.md#running-via-docker) fails when execute:

```
docker run -v $HOME/prysm-data:/data -p 4000:4000 \
  gcr.io/prysmaticlabs/prysm/beacon-chain:latest \
  --name beacon-node \
  --datadir=/data
```

You should put `--name` after port configuration. Currently, the result of executing this command is:

```
time="2019-10-28T10:46:48Z" level=error msg="Failed to calculate roughtime offset" error="no valid responses" prefix=roughtime
2019/10/28 10:46:48 maxprocs: Leaving GOMAXPROCS=2: CPU quota undefined
Incorrect Usage. flag provided but not defined: -name

NAME:
   beacon-chain - this is a beacon chain implementation for Ethereum 2.0
USAGE:
   image.binary [options] command [command options] [arguments...]
```

**Comment by fooock** [comment]
Related to #3692

**Comment by har00ga** [comment]
Thanks for noting this! Adjustments have been merged with #3834
